web/operator/dist/build.js

```diff
@@ -1,9 +1,16 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
-var xmlHttp = new XMLHttpRequest();
+//var xmlHttp = new XMLHttpRequest();
 
-xmlHttp.open("GET", "/api/dashboard?satelliteId=12D1kqUXtJiCsS72UKeCGHyKSQy1FSonerZ5fb6nSQAaSim4Vag", false );
-xmlHttp.send();
+//xmlHttp.open("GET", "/api/dashboard?satelliteId=12D1kqUXtJiCsS72UKeCGHyKSQy1FSonerZ5fb6nSQAaSim4Vag", false);
+//xmlHttp.send();
 
-alert(xmlHttp.responseText);
\ No newline at end of file
+var ws = new WebSocket("ws://localhost:14002/api/update/");
```

Comment:
hardcoded adress

Reply:
diat is added only for testing purposes and as an example
it will be removed  by pull request of @NikolaiYurchenko and @VitaliiShpital

Reply:
Can we add the corresponding ToDo's here then?
Also are the other commented notes ever being used again?